framework/test/unit/engine/consensus/certificate_generation/commit_pool.spec.ts

```diff
@@ -1380,5 +1410,19 @@ describe('CommitPool', () => {
 				certificateSignature: Buffer.alloc(0),
 			});
 		});
+
+		it('should certify the minCertifyHeight even when heightNextBFTParameters is lower', async () => {
```

Comment:
I am not sure I understand what you mean here, since you defined 

`heightNextBFTParameters =  Math.max(heightNextBFTParameters, this._minCertifyHeight + 1)`

So I guess you mean "should certify the minCertifyHeight even when getNextHeightBFTParameters returns a lower value than `MIN_CERTIFICATE_HEIGHT + 1`". 

If this is the case, I would suggest to be careful and try both cases:

i ) `getNextHeightBFTParameters < MIN_CERTIFICATE_HEIGHT`
ii) `getNextHeightBFTParameters = MIN_CERTIFICATE_HEIGHT` (should accept again and this is the important case where the previous version had the problem of rejection)

Reply:
Not sure if the comment of @janhack covers the same case.

Reply:
Added the tests 👍